### ISSUE TITLE
ILogger provider does not use TelemetryConfiguration with OperationCorrelationTelemetryInitializer

### ISSUE THREAD

**Issue by cijothomas** [microsoft/ApplicationInsights-dotnet#1455]
ApplicationInsightsLoggerProvider creates a new TelemetryConfiguration which do not contain any initializers.  It should use TelemetryConfiguration.CreateDefault() so that OperationCorrelationTelemetryInitializer is added by default.
Without this, the logs from Ilogger will not get correlation with parent operations, if any. (Typically occurs if logs are made within a telemetryClient.StartOperation block)

**Comment by cijothomas** [comment]
The recommendation is to use WorkerService SDK or Asp.Net Core SDK based on application type. In both these cases, the reported issue will not occur.

It still occurs if one just uses LoggerProvider standalone - but in those cases, everything is manually collected like Requests etc, so OperationCOrrelationinitializer can be added manually as well. 

Closing this issue as no longer relevant/important.

**Comment by supereddie** [comment]
> The recommendation is to use WorkerService SDK or Asp.Net Core SDK based on application type. In both these cases, the reported issue will not occur.
> 
> It still occurs if one just uses LoggerProvider standalone - but in those cases, everything is manually collected like Requests etc, so OperationCOrrelationinitializer can be added manually as well.
> 
> Closing this issue as no longer relevant/important.

Hi @cijothomas, 
Apologies for resurrecting such an old issue, however I've just ran into this one. We are unfortunately still using the old ASP.NET with .NET Framework 4.7 and converting it to a newer .NET version is going very slow.

Application Insights itself - with request and dependency collectors and stuff - is working properly, with the configuration loaded from applicationinsights.config.

We manually added DI which is working great, but not for ILogger / ApplicationInsightsLoggerProvider. More specifically: none of the ILogger.LogXXX0-methods generate telemetry which is properly correlated to the Operation. The telemetry itself gets send properly to AI, but it is just not correlated (of has any of the other telemetry initializers applied).

Manually adding the TelemetryInitializers (as defined in applicationinsights.config) to the IOptions-instance is working, but is very tedious and error-prone. 

It would be very helpful if there was an easier way to load settings from the 'default' places - similar to what `TelemetryConfiguration.Active` does but on an existing instance. Perhaps an extension-method or something on TelemetryConfiguration - specifically for older .NET Framework applications? So I could do e.g.
```csharp
services.AddOptions<TelemetryConfiguration>().Configure(c => c.LoadDefaults());
services.AddLogging(b => b.AddApplicationInsights());
````

We have a small number of ASP.NET sites still using .NET Framework 4.7/8 so it would be very helpful if there was something similar for easier diagnostics.
